src/conversions/index.js

```diff
@@ -1,17 +1,33 @@
-module.exports = {
-  hex: require('./fromHex.js'),
-  rgb: require('./fromRgb.js'),
-  cssrgb: require('./fromCssRgb.js'),
-  hsl: require('./fromHsl.js'),
-  csshsl: require('./fromCssHsl.js'),
-  hsv: require('./fromHsv.js'),
-  cmyk: require('./fromCmyk.js'),
-  yiq: require('./fromYiq.js'),
-  XYZ: require('./fromXYZ.js'),
-  xyY: require('./fromxyY.js'),
-  lms: require('./fromLms.js'),
-  cielab: require('./fromCieLab.js'),
-  cieluv: require('./fromCieLuv.js'),
-  cielch: require('./fromCieLch.js'),
-  hsluv: require('./fromHsluv.js')
+import cielab from './cielab'
```

Comment:
It is actually: https://github.com/toish/chromatism/blob/refactor/src/helpers/convert-to-type.js#L2